src/core/public/notifications/banners/global_banner_list.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import * as Rx from 'rxjs';
+
+import { Banners } from './banners_service';
+import './global_banner_list.css';
+
+interface Props {
+  banners$: Rx.Observable<Banners>;
+}
+
+interface State {
+  error?: Error;
+  banners?: Banners;
+}
+
+export class GlobalBannerList extends React.Component<Props> {
+  public state: State = {};
+  private subscription?: Rx.Subscription;
+
+  public componentDidMount() {
+    this.subscription = this.props.banners$.subscribe({
+      next: banners => {
+        this.setState({ banners });
+      },
+      error: error => {
+        this.setState({ error });
+      },
+    });
+  }
+
+  public componentWillUnmount() {
+    if (this.subscription) {
+      this.subscription.unsubscribe();
+    }
+  }
+
+  public render() {
+    const { error, banners } = this.state;
+
+    if (error) {
+      throw error;
+    }
+
+    if (!banners || !banners.length) {
+      return null;
+    }
+
+    return (
+      <div className="globalBanner__list">
+        {banners.map(({ id, component, priority, ...rest }) => (
+          <div key={id} className="globalBanner__item" {...rest}>
```

Comment:
I don't think so, the banners themselves are really what I think we should focus on moving to EUI, but as you see here they are passed in fully styled. I tried moving the divs here to use `<EuiFlexGroup />`, but since we need refs we have issues there, so I didn't think it was worth the effort right now.